### ISSUE TITLE
Tagging errors for ADP/SCONJ

### ISSUE THREAD

**Issue by muchang** [UniversalDependencies/UD_English-GUM#29]
Here are some suspicious cases that should be SCONJ while they are tagged as ADP in GUM.

Word: "during".
Reason: The "during" in the term "during doing" seems to be tagged as SCONJ, such as in "GUM_academic_theropod-6".
```
# sent_id = GUM_academic_theropod-32
# s_type = decl
# text = Interestingly, the femur kinematics during walking in our control-weight group resembles the results reported in the experimental subjects of Carrano and Biewener.
1	Interestingly	interestingly	ADV	RB	Degree=Pos	12	advmod	12:advmod	Discourse=evidence:103->106|SpaceAfter=No
2	,	,	PUNCT	,	_	1	punct	1:punct	_
3	the	the	DET	DT	Definite=Def|PronType=Art	5	det	5:det	Entity=(abstract-165(abstract-166
4	femur	femur	NOUN	NN	Number=Sing	5	compound	5:compound	Entity=(object-91)
5	kinematics	kinematics	NOUN	NN	Number=Sing	12	nsubj	12:nsubj	Entity=abstract-166)
6	during	during	ADP	IN	_	7	case	7:case	_
7	walking	walking	NOUN	NN	Number=Sing	5	nmod	5:nmod:during	Entity=(event-25)
8	in	in	ADP	IN	_	11	case	11:case	_
9	our	our	PRON	PRP$	Number=Plur|Person=1|Poss=Yes|PronType=Prs	11	nmod:poss	11:nmod:poss	Entity=(animal-36(person-2)
10	control-weight	control-weight	NOUN	NN	Number=Sing	11	compound	11:compound	_
11	group	group	NOUN	NN	Number=Sing	5	nmod	5:nmod:in	Entity=animal-36)
12	resembles	resemble	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
13	the	the	DET	DT	Definite=Def|PronType=Art	14	det	14:det	Entity=(abstract-126
14	results	result	NOUN	NNS	Number=Plur	12	obj	12:obj	_
15	reported	report	VERB	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	14	acl	14:acl	Discourse=elaboration:104->103
16	in	in	ADP	IN	_	19	case	19:case	_
17	the	the	DET	DT	Definite=Def|PronType=Art	19	det	19:det	Entity=(animal-134
18	experimental	experimental	ADJ	JJ	Degree=Pos	19	amod	19:amod	_
19	subjects	subject	NOUN	NNS	Number=Plur	15	obl	15:obl:in	_
20	of	of	ADP	IN	_	21	case	21:case	_
21	Carrano	Carrano	PROPN	NNP	Number=Sing	19	nmod	19:nmod:of	Entity=(person-124(person-40)
22	and	and	CCONJ	CC	_	23	cc	23:cc	_
23	Biewener	Biewener	PROPN	NNP	Number=Sing	21	conj	19:nmod:of|21:conj:and	Entity=(person-41)abstract-165)abstract-126)animal-134)person-124)|SpaceAfter=No
24	.	.	PUNCT	.	_	12	punct	12:punct	_
```

Word: "of" (second).
Reason: The "of" in the term "of what" seems to be tagged as SCONJ, such as in "GUM_voyage_fortlee-40".
```
# sent_id = GUM_bio_fillmore-43
# s_type = decl
# text = There, he started a project called FrameNet, an on-line structured description of the English lexicon implementing much of what he had earlier proposed more theoretically in his theory of Frame semantics, while implementing the idea of emphasizing example sentences from corpora.
1	There	there	ADV	RB	PronType=Dem	4	advmod	4:advmod	Discourse=joint:102->17|SpaceAfter=No
2	,	,	PUNCT	,	_	1	punct	1:punct	_
3	he	he	PRON	PRP	Case=Nom|Gender=Masc|Number=Sing|Person=3|PronType=Prs	4	nsubj	4:nsubj	Entity=(person-1-Charles_J._Fillmore)
4	started	start	VERB	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	0	root	0:root	_
5	a	a	DET	DT	Definite=Ind|PronType=Art	6	det	6:det	Entity=(abstract-183-FrameNet
6	project	project	NOUN	NN	Number=Sing	4	obj	4:obj	_
7	called	call	VERB	VBN	Tense=Past|VerbForm=Part	6	acl	6:acl	_
8	FrameNet	FrameNet	PROPN	NNP	Number=Sing	7	xcomp	7:xcomp	Bridge=abstract-183-FrameNet<abstract-206-FrameNet|Entity=(abstract-206-FrameNet)abstract-183-FrameNet)|SpaceAfter=No
9	,	,	PUNCT	,	_	13	punct	13:punct	_
10	an	a	DET	DT	Definite=Ind|PronType=Art	13	det	13:det	Entity=(abstract-206-FrameNet
11	on-line	on-line	ADJ	JJ	Degree=Pos	13	amod	13:amod	_
12	structured	structure	VERB	VBN	Tense=Past|VerbForm=Part	13	amod	13:amod	_
13	description	description	NOUN	NN	Number=Sing	8	appos	8:appos	_
14	of	of	ADP	IN	_	17	case	17:case	_
15	the	the	DET	DT	Definite=Def|PronType=Art	17	det	17:det	Entity=(abstract-207
16	English	English	ADJ	JJ	Degree=Pos	17	compound	17:compound	_
17	lexicon	lexicon	NOUN	NN	Number=Sing	13	nmod	13:nmod:of	Entity=abstract-207)
18	implementing	implement	VERB	VBG	VerbForm=Ger	13	acl	13:acl	Discourse=elaboration:103->102
19	much	much	ADJ	JJ	Degree=Pos	18	obj	18:obj	Entity=(abstract-208
20	of	of	ADP	IN	_	21	case	21:case	_
21	what	what	PRON	WP	PronType=Rel	19	nmod	19:nmod:of	_
22	he	he	PRON	PRP	Case=Nom|Gender=Masc|Number=Sing|Person=3|PronType=Prs	25	nsubj	25:nsubj	Entity=(person-1-Charles_J._Fillmore)
23	had	have	AUX	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	25	aux	25:aux	_
24	earlier	early	ADV	RBR	Degree=Cmp	25	advmod	25:advmod	_
25	proposed	propose	VERB	VBN	Tense=Past|VerbForm=Part	21	acl:relcl	21:acl:relcl	_
26	more	more	ADV	RBR	Degree=Cmp	27	advmod	27:advmod	_
27	theoretically	theoretically	ADV	RB	Degree=Pos	25	advmod	25:advmod	_
28	in	in	ADP	IN	_	30	case	30:case	_
29	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	30	nmod:poss	30:nmod:poss	Entity=(abstract-209(person-1-Charles_J._Fillmore)
30	theory	theory	NOUN	NN	Number=Sing	25	obl	25:obl:in	_
31	of	of	ADP	IN	_	33	case	33:case	_
32	Frame	frame	NOUN	NN	Number=Sing	33	compound	33:compound	Entity=(abstract-96-Frame_Semantics
33	semantics	semantics	NOUN	NN	Number=Sing	30	nmod	30:nmod:of	Entity=abstract-206-FrameNet)abstract-208)abstract-209)abstract-96-Frame_Semantics)|SpaceAfter=No
34	,	,	PUNCT	,	_	36	punct	36:punct	_
35	while	while	SCONJ	IN	_	36	mark	36:mark	Discourse=joint:104->103
36	implementing	implement	VERB	VBG	VerbForm=Ger	4	advcl	4:advcl:while	_
37	the	the	DET	DT	Definite=Def|PronType=Art	38	det	38:det	Entity=(abstract-210
38	idea	idea	NOUN	NN	Number=Sing	36	obj	36:obj	_
39	of	of	SCONJ	IN	_	40	mark	40:mark	Discourse=elaboration:105->104
40	emphasizing	emphasize	VERB	VBG	VerbForm=Ger	38	acl	38:acl:of	_
41	example	example	NOUN	NN	Number=Sing	42	compound	42:compound	Entity=(abstract-199
42	sentences	sentence	NOUN	NNS	Number=Plur	40	obj	40:obj	_
43	from	from	ADP	IN	_	44	case	44:case	_
44	corpora	corpus	NOUN	NNS	Number=Plur	42	nmod	42:nmod:from	Entity=(abstract-200)abstract-210)abstract-199)|SpaceAfter=No
45	.	.	PUNCT	.	_	4	punct	4:punct	_
```

Word: "of" (second).
Reason: Similar to above
```
# sent_id = GUM_interview_daly-8
# s_type = decl
# speaker = MarymeganDaly
# text = The technology that Frank and his colleagues are using to explore the ice is so important because, given our lack of data, we have no reasonable expectation of what it should be like, or what it shouldn't be like.
1	The	the	DET	DT	Definite=Def|PronType=Art	2	det	2:det	Discourse=result:15->19|Entity=(object-21
2	technology	technology	NOUN	NN	Number=Sing	16	nsubj	9:obj|16:nsubj	_
3	that	that	PRON	WDT	PronType=Rel	9	obj	2:ref	Discourse=elaboration:16->15
4	Frank	Frank	PROPN	NNP	Number=Sing	9	nsubj	9:nsubj	Entity=(person-10(person-9)
5	and	and	CCONJ	CC	_	7	cc	7:cc	_
6	his	his	PRON	PRP$	Gender=Masc|Number=Sing|Person=3|Poss=Yes|PronType=Prs	7	nmod:poss	7:nmod:poss	Entity=(person-22(person-9)
7	colleagues	colleague	NOUN	NNS	Number=Plur	4	conj	4:conj:and|9:nsubj	Entity=person-10)person-22)
8	are	be	AUX	VBP	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	9	aux	9:aux	_
9	using	use	VERB	VBG	Tense=Pres|VerbForm=Part	2	acl:relcl	2:acl:relcl	_
10	to	to	PART	TO	_	11	mark	11:mark	Discourse=purpose:17->16
11	explore	explore	VERB	VB	VerbForm=Inf	9	xcomp	9:xcomp	_
12	the	the	DET	DT	Definite=Def|PronType=Art	13	det	13:det	Entity=(place-15
13	ice	ice	NOUN	NN	Number=Sing	11	obj	11:obj	Entity=object-21)place-15)
14	is	be	AUX	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	16	cop	16:cop	Discourse=same-unit:18->15
15	so	so	ADV	RB	_	16	advmod	16:advmod	_
16	important	important	ADJ	JJ	Degree=Pos	0	root	0:root	_
17	because	because	SCONJ	IN	_	26	mark	26:mark	Discourse=elaboration:19->14|SpaceAfter=No
18	,	,	PUNCT	,	_	17	punct	17:punct	_
19	given	give	VERB	VBN	Tense=Past|VerbForm=Part	21	case	21:case	Discourse=circumstance:20->21
20	our	our	PRON	PRP$	Number=Plur|Person=1|Poss=Yes|PronType=Prs	21	nmod:poss	21:nmod:poss	Entity=(abstract-23(person-24)
21	lack	lack	NOUN	NN	Number=Sing	26	obl	26:obl:given	_
22	of	of	ADP	IN	_	23	case	23:case	_
23	data	datum	NOUN	NNS	Number=Plur	21	nmod	21:nmod:of	Entity=abstract-23)|SpaceAfter=No
24	,	,	PUNCT	,	_	21	punct	21:punct	_
25	we	we	PRON	PRP	Case=Nom|Number=Plur|Person=1|PronType=Prs	26	nsubj	26:nsubj	Discourse=same-unit:21->19
26	have	have	VERB	VBP	Mood=Ind|Number=Plur|Person=1|Tense=Pres|VerbForm=Fin	16	advcl	16:advcl:because	_
27	no	no	DET	DT	Polarity=Neg	29	det	29:det	Entity=(abstract-25
28	reasonable	reasonable	ADJ	JJ	Degree=Pos	29	amod	29:amod	_
29	expectation	expectation	NOUN	NN	Number=Sing	26	obj	26:obj	_
30	of	of	ADP	IN	_	31	case	31:case	Discourse=elaboration:22->21
31	what	what	PRON	WP	PronType=Rel	29	nmod	29:nmod:of	_
32	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	35	nsubj	35:nsubj	Entity=(abstract-14)
33	should	should	AUX	MD	VerbForm=Fin	35	aux	35:aux	_
34	be	be	AUX	VB	VerbForm=Inf	35	cop	35:cop	_
35	like	like	ADJ	JJ	Degree=Pos	31	acl:relcl	31:acl:relcl	SpaceAfter=No
36	,	,	PUNCT	,	_	43	punct	43:punct	_
37	or	or	CCONJ	CC	_	43	cc	43:cc	Discourse=contrast:23->22
38	what	what	PRON	WP	PronType=Int	43	obj	43:obj	_
39	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	43	nsubj	43:nsubj	Entity=(abstract-14)
40-41	shouldn't	_	_	_	_	_	_	_	_
40	should	should	AUX	MD	VerbForm=Fin	43	aux	43:aux	_
41	n't	not	PART	RB	Polarity=Neg	43	advmod	43:advmod	_
42	be	be	AUX	VB	VerbForm=Inf	43	cop	43:cop	_
43	like	like	ADJ	JJ	Degree=Pos	30	conj	30:conj:or|31:case	Entity=abstract-25)|SpaceAfter=No
44	.	.	PUNCT	.	_	16	punct	16:punct	_
```

Commit: d38df82

**Comment by nschneid** [comment]
In general, ADP tokens should attach as `case` to a NOUN, while SCONJ tokens should attach as `mark` to a VERB. A lot of words are ambiguous between ADP and SCONJ. So I'm not sure there's an inconsistency.

**Comment by muchang** [comment]
Thanks, Nathan. With this standard, the other two cases would be wrong, i.e., tagging ADP as SCONJ.

Word: "during"
Reason: Attaching a NOUN (not VERB) as `mark`
```
# sent_id = GUM_academic_theropod-6
# s_type = decl
# text = In fact, the control-weight subjects showed a more horizontally oriented femur during walking with respect to the control group, similar to that observed in Carrano and Biewener's experimental subjects [7].
1	In	in	ADP	IN	_	2	case	2:case	Discourse=elaboration:15->13
2	fact	fact	NOUN	NN	Number=Sing	7	obl	7:obl:in	SpaceAfter=No
3	,	,	PUNCT	,	_	2	punct	2:punct	_
4	the	the	DET	DT	Definite=Def|PronType=Art	6	det	6:det	Entity=(animal-36
5	control-weight	control-weight	NOUN	NN	Number=Sing	6	compound	6:compound	_
6	subjects	subject	NOUN	NNS	Number=Plur	7	nsubj	7:nsubj	Entity=animal-36)
7	showed	show	VERB	VBD	Mood=Ind|Number=Plur|Person=3|Tense=Past|VerbForm=Fin	0	root	0:root	_
8	a	a	DET	DT	Definite=Ind|PronType=Art	12	det	12:det	Entity=(object-37
9	more	more	ADV	RBR	Degree=Cmp	10	advmod	10:advmod	_
10	horizontally	horizontally	ADV	RB	Degree=Pos	11	advmod	11:advmod	_
11	oriented	orient	VERB	VBN	Tense=Past|VerbForm=Part	12	amod	12:amod	_
12	femur	femur	NOUN	NN	Number=Sing	7	obj	7:obj	Entity=object-37)
13	during	during	SCONJ	IN	_	14	mark	14:mark	_
14	walking	walking	NOUN	NN	Number=Sing	7	advcl	7:advcl:during	Entity=(event-25)
15	with	with	ADP	IN	_	16	case	16:case	_
16	respect	respect	NOUN	NN	Number=Sing	7	obl	7:obl:with	_
17	to	to	ADP	IN	_	20	case	20:case	_
18	the	the	DET	DT	Definite=Def|PronType=Art	20	det	20:det	Entity=(animal-21
19	control	control	NOUN	NN	Number=Sing	20	compound	20:compound	_
20	group	group	NOUN	NN	Number=Sing	16	nmod	16:nmod:to	Entity=animal-21)|SpaceAfter=No
21	,	,	PUNCT	,	_	22	punct	22:punct	_
22	similar	similar	ADJ	JJ	Degree=Pos	7	advmod	7:advmod	_
23	to	to	ADP	IN	_	24	case	24:case	_
24	that	that	PRON	DT	Number=Sing|PronType=Dem	22	obl	22:obl:to	Bridge=object-37<object-38|Entity=(object-38
25	observed	observe	VERB	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	24	acl	24:acl	Discourse=elaboration:16->15
26	in	in	ADP	IN	_	32	case	32:case	_
27	Carrano	Carrano	PROPN	NNP	Number=Sing	32	nmod:poss	32:nmod:poss	Entity=(animal-39(person-40)
28	and	and	CCONJ	CC	_	29	cc	29:cc	_
29-30	Biewener's	_	_	_	_	_	_	_	_
29	Biewener	Biewener	PROPN	NNP	Number=Sing	27	conj	27:conj:and|32:nmod:poss	Entity=(person-41
30	's	's	PART	POS	_	27	case	27:case	Entity=person-41)
31	experimental	experimental	ADJ	JJ	Degree=Pos	32	amod	32:amod	_
32	subjects	subject	NOUN	NNS	Number=Plur	25	obl	25:obl:in	Entity=object-38)animal-39)
33	[	[	PUNCT	-LRB-	_	34	punct	34:punct	Discourse=evidence:17->16|SpaceAfter=No
34	7	7	NUM	CD	NumForm=Digit|NumType=Card	24	dep	24:dep	Entity=(abstract-42)|SpaceAfter=No
35	]	]	PUNCT	-RRB-	_	34	punct	34:punct	SpaceAfter=No
36	.	.	PUNCT	.	_	7	punct	7:punct	_
```

Word: "of"
Reason: Attaching a PRON (not VERB) as `mark`
```
# sent_id = GUM_voyage_fortlee-40
# s_type = decl
# text = Audio-visual displays, detailed exhibits, gives visitors a sense of what this town was like centuries ago.
1	Audio-visual	audio-visual	ADJ	JJ	Degree=Pos	2	amod	2:amod	Bridge=abstract-128<object-130|Discourse=elaboration:80->79|Entity=(object-130
2	displays	display	NOUN	NNS	Number=Plur	7	nsubj	7:nsubj	Entity=object-130)|SpaceAfter=No
3	,	,	PUNCT	,	_	5	punct	5:punct	_
4	detailed	detailed	ADJ	JJ	Degree=Pos	5	amod	5:amod	Bridge=abstract-128<object-131|Entity=(object-131
5	exhibits	exhibit	NOUN	NNS	Number=Plur	2	conj	2:conj|7:nsubj	Entity=object-131)|SpaceAfter=No
6	,	,	PUNCT	,	_	2	punct	2:punct	_
7	gives	give	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
8	visitors	visitor	NOUN	NNS	Number=Plur	7	iobj	7:iobj	Entity=(person-132)
9	a	a	DET	DT	Definite=Ind|PronType=Art	10	det	10:det	Entity=(abstract-133
10	sense	sense	NOUN	NN	Number=Sing	7	obj	7:obj	_
11	of	of	SCONJ	IN	_	12	mark	12:mark	_
12	what	what	PRON	WP	PronType=Rel	10	acl	10:acl:of	_
13	this	this	DET	DT	Number=Sing|PronType=Dem	14	det	14:det	Entity=(place-1-Fort_Lee%2C_New_Jersey
14	town	town	NOUN	NN	Number=Sing	16	nsubj	16:nsubj	Entity=place-1-Fort_Lee%2C_New_Jersey)
15	was	be	AUX	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	16	cop	16:cop	_
16	like	like	ADP	IN	_	12	acl:relcl	12:acl:relcl	_
17	centuries	century	NOUN	NNS	Number=Plur	18	obl:tmod	18:obl:tmod	Entity=(time-134)
18	ago	ago	ADV	RB	Degree=Pos	16	advmod	16:advmod	Entity=abstract-133)|SpaceAfter=No
19	.	.	PUNCT	.	_	7	punct	7:punct	_
```

**Comment by amir-zeldes** [comment]
I agree GUM_academic_theropod-6 and GUM_voyage_fortlee-40 are errors. For -ing forms, there are tests based on PTB guidelines, for example can we insert an article ("the walking") or adjective ("fast walking") for NOUN and adverb for VERB ("walking quickly"). The WH pronoun case is more clear-cut, since these are free relatives, and the WH pronoun serves as an NP head and is not considered verbal in any way. 

In GUM_voyage_fortlee-40 we have an unusual free relative in that the WH pronoun is the fronted governor of "like" ("it was like what?"), but because we only have one instance of "what", the "like" has to stay in the subordinate clause. This means that the tree is wrong - I think it should be nmod + case:

```CoNLL-U
1	Audio-visual	audio-visual	ADJ	JJ	Degree=Pos	2	amod	_	Bridge=abstract-128<object-130|Discourse=elaboration:80->79|Entity=(object-130
2	displays	display	NOUN	NNS	Number=Plur	7	nsubj	_	Entity=object-130)|SpaceAfter=No
3	,	,	PUNCT	,	_	5	punct	_	_
4	detailed	detailed	ADJ	JJ	Degree=Pos	5	amod	_	Bridge=abstract-128<object-131|Entity=(object-131
5	exhibits	exhibit	NOUN	NNS	Number=Plur	2	conj	7:nsubj	Entity=object-131)|SpaceAfter=No
6	,	,	PUNCT	,	_	2	punct	_	_
7	gives	give	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	_	_
8	visitors	visitor	NOUN	NNS	Number=Plur	7	iobj	_	Entity=(person-132)
9	a	a	DET	DT	Definite=Ind|PronType=Art	10	det	_	Entity=(abstract-133
10	sense	sense	NOUN	NN	Number=Sing	7	obj	_	_
11	of	of	ADP	IN	_	12	case	_	_
12	what	what	PRON	WP	PronType=Rel	10	nmod	_	_
13	this	this	DET	DT	Number=Sing|PronType=Dem	14	det	_	Entity=(place-1-Fort_Lee%2C_New_Jersey
14	town	town	NOUN	NN	Number=Sing	16	nsubj	_	Entity=place-1-Fort_Lee%2C_New_Jersey)
15	was	be	AUX	VBD	Mood=Ind|Number=Sing|Person=3|Tense=Past|VerbForm=Fin	16	cop	_	_
16	like	like	ADP	IN	_	12	acl:relcl	_	_
17	centuries	century	NOUN	NNS	Number=Plur	18	obl:tmod	_	Entity=(time-134)
18	ago	ago	ADV	RB	Degree=Pos	16	advmod	_	Entity=abstract-133)|SpaceAfter=No
19	.	.	PUNCT	.	_	7	punct	_	_
```

So basically we are promoting "like" where the reconstruction of the free relative is something like:

  * Audio-visual displays, detailed exhibits, gives visitors a sense of **that which** this town was like centuries ago.

**Comment by muchang** [comment]
Thanks, Amir. Here is an additional suspicious case that is tagging ADP as SCONJ.

Word: "in" (second)
Reason: Attaching a NOUN (not VERB) as `mark`
```
# sent_id = GUM_academic_theropod-11
# s_type = sub
# text = Differences in limb orientation can produce substantial differences in loading regimes on limb bones.
1	Differences	difference	NOUN	NNS	Number=Plur	6	nsubj	6:nsubj	Discourse=background:31->54|Entity=(abstract-57
2	in	in	ADP	IN	_	4	case	4:case	_
3	limb	limb	NOUN	NN	Number=Sing	4	compound	4:compound	Entity=(abstract-9(object-44)
4	orientation	orientation	NOUN	NN	Number=Sing	1	nmod	1:nmod:in	Entity=abstract-57)abstract-9)
5	can	can	AUX	MD	VerbForm=Fin	6	aux	6:aux	_
6	produce	produce	VERB	VB	VerbForm=Inf	0	root	0:root	_
7	substantial	substantial	ADJ	JJ	Degree=Pos	8	amod	8:amod	Entity=(abstract-58
8	differences	difference	NOUN	NNS	Number=Plur	6	obj	6:obj	_
9	in	in	SCONJ	IN	_	11	mark	11:mark	_
10	loading	loading	NOUN	NN	Number=Sing	11	compound	11:compound	Entity=(abstract-59
11	regimes	regime	NOUN	NNS	Number=Plur	8	nmod	8:nmod	Entity=abstract-59)
12	on	on	ADP	IN	_	14	case	14:case	_
13	limb	limb	NOUN	NN	Number=Sing	14	compound	14:compound	Entity=(object-60(object-44)
14	bones	bone	NOUN	NNS	Number=Plur	8	nmod	8:nmod:on	Entity=abstract-58)object-60)|SpaceAfter=No
15	.	.	PUNCT	.	_	6	punct	6:punct	_
```

**Comment by nschneid** [comment]
Here's a query that turns up a number of issues in GUM regarding SCONJ attaching to a NOUN: http://match.grew.fr/?corpus=UD_English-GUM@dev&custom=61015a78700df&clustering=N.upos

**Comment by amir-zeldes** [comment]
Thanks @muchang GUM_academic_theropod-11 is definitely an error.

@nschneid thanks, there were a lot of errors there, though some I think are correct (ellipses, other weird things happening). The rest should be fixed upstream now.
